indico/modules/events/abstracts/controllers/email_templates.py

```diff
@@ -33,7 +36,14 @@ class RHEmailTemplateList(RHManageAbstractsBase):
     """Display list of e-mail templates."""
 
     def _process(self):
-        return jsonify_template('events/abstracts/management/notification_tpl_list.html', event=self.event_new)
+        self.commit = False
+        track_dict, abstract_states, contr_type_dict, email_tpl_dict = _get_rules_fields(self.event_new)
+        return jsonify_template('events/abstracts/management/notification_tpl_list.html',
+                                event=self.event_new,
```

Comment:
True, just keep putting arguments in the same line until it reaches the barrier of 120 chars, then go to a new line.